kotlin_client/src/main/java/com/github/paylike/kotlin_client/domain/payment/response/PaymentResponse.kt

```diff
@@ -0,0 +1,49 @@
+package com.github.paylike.kotlin_client.domain.payment.response
+
+import com.github.paylike.kotlin_client.domain.payment.response.challenges.Challenge
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.JsonObject
+
+/**
+ * Response types to payment requests
+ *
+ * One of :
+ * [challenges],
+ * or
+ * [authorizationId]
+ * or
+ * [hints]
+ * or
+ * [hints], // optional
+ * [action],
+ * [fields],
+ * [timeout], // optional
+ */
+@Serializable
+data class PaymentResponse(
+    /**
+     * Challenges type of response has this field
+     */
+    val challenges: List<Challenge>? = null,
+
+    /**
+     * Final response has this field
+     */
+    val authorizationId: String? = null,
+    val transactionId: String? = null, // TODO is this existing?
```

Comment:
It does, but I am not sure that you can get `transactionId` from payment creation. It's a bit of an inconsitency in the API though

Reply:
Then I left it there in case it came.